sockeye/utils.py

```diff
@@ -241,45 +241,41 @@ def variance(self) -> float:
         else:
             return self._M2 / self._count
 
-
-def smallest_k(matrix: np.ndarray, k: int,
-               only_first_row: bool = False) -> Tuple[Tuple[np.ndarray, np.ndarray], np.ndarray]:
+def topk(scores: mx.nd.NDArray, k: int, batch_size: int, t: int, use_mxnet_topk: bool):
```

Comment:
can we add typing for the return value, i.e. `Tuple[np.ndarray, np.ndarray, Union[np.ndarray, mx.nd.NDArray]]` ?